app/controllers/friendships_controller.rb

```diff
@@ -11,6 +16,5 @@ def new
     else
       user.add_friend(new_friend)
     end
-    redirect_to '/dashboard'
```

Comment:
unsure if we need this.

Reply:
The method was just two lines too long, so in order for rubo cop to like it I had to do this